app-ios/Sources/ContributorFeature/ContributorView.swift

```diff
@@ -1,27 +1,119 @@
 import SwiftUI
 import ComposableArchitecture
+import Model
+import Assets
+import Theme
+import NukeUI
 
 public struct ContributorState: Equatable {
-    public init() {}
+    public var contributors: [Contributor]
+
+    public init(contributors: [Contributor] = []) {
+        self.contributors = contributors
+    }
 }
 
-public enum ContributorAction {}
+public enum ContributorAction {
+    case refresh
+    case refreshResponse(TaskResult<[Contributor]>)
+}
 public struct ContributorEnvironment {
-    public init() {}
+    public let contributorsRepository: ContributorsRepository
+
+    public init(contributorsRepository: ContributorsRepository) {
+        self.contributorsRepository = contributorsRepository
+    }
 }
-public let contributorReducer = Reducer<ContributorState, ContributorAction, ContributorEnvironment> { _, _, _ in
-    return .none
+public let contributorReducer = Reducer<ContributorState, ContributorAction, ContributorEnvironment> { state, action, environment in
+    switch action {
+    case .refresh:
+        return .run { @MainActor subscriber in
+            for try await result: [Contributor] in environment.contributorsRepository.contributors().stream() {
+                await subscriber.send(
+                    .refreshResponse(
+                        TaskResult {
+                            result
+                        }
+                    )
+                )
+            }
+        }
+        .receive(on: DispatchQueue.main.eraseToAnyScheduler())
+        .eraseToEffect()
+    case .refreshResponse(.success(let contributors)):
+        state.contributors = contributors
+        return .none
+    case .refreshResponse(.failure):
+        return .none
+    }
 }
 
 public struct ContributorView: View {
     private let store: Store<ContributorState, ContributorAction>
 
     public init(store: Store<ContributorState, ContributorAction>) {
         self.store = store
+
+        UITabBar.appearance().barTintColor = AssetColors.surface.color
     }
 
     public var body: some View {
-        Text("TODO: Contributor")
+        WithViewStore(store) { viewStore in
+            ScrollView {
+                LazyVStack(spacing: 0) {
```

Comment:
Using "LazyVStack" instead of "List" due to disappear Nuke's "LazyImage" (Kingfisher's KFImage also) when scrolling.